src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -48,18 +49,26 @@ def check_config(self):
         raise InvalidPluginConfiguration("Invalid")
 
 
-@patch(
-    "openforms.emails.tasks.GlobalConfiguration.get_solo",
-    return_value=GlobalConfiguration(
-        recipients_email_digest=["tralala@test.nl", "trblblb@test.nl"]
-    ),
-)
 @override_settings(LANGUAGE_CODE="en")
 class EmailDigestTaskIntegrationTests(TestCase):
 
+    def setUp(self) -> None:
+        self.config = GlobalConfiguration.get_solo()
+        self.config.recipients_email_digest = ["tralala@test.nl", "trblblb@test.nl"]
+        self.config.save()
+
+        self.referentielijsten_service = ServiceFactory.create(
+            api_root="http://localhost:8004/api/v1/",
+            slug="referentielijsten",
+            auth_type=AuthTypes.no_auth,
+        )
+        self.config.referentielijsten_services.add(self.referentielijsten_service)
+
+    @patch("openforms.emails.tasks.GlobalConfiguration.get_solo")
     def test_that_repeated_failures_are_not_mentioned_multiple_times(
         self, mock_global_config
     ):
+        mock_global_config.return_value = self.config
```

Comment:
mocking the config is now pointless here as we are updating the actual real config in tests anyway. The mocking is a mechanism to ensure we don't accidentally mess with the global django-solo cache. Best to just remove the mocks here.